learners/nonuniform_quantization/bit_optimizer.py

```diff
@@ -278,7 +278,8 @@ def __layerwise_finetune(self, feed_dict_train, layer_bits):
   def __global_finetune(self, feed_dict_train):
     time_prev = timer()
     for t_step in range(self.tune_global_steps):
-      _ = self.sess_train.run(self.ops['train'], feed_dict=feed_dict_train)
+      # _ = self.sess_train.run(self.ops['train'], feed_dict=feed_dict_train)
+      _ = self.sess_train.run(self.ops['rl_fintune'], feed_dict=feed_dict_train)
```

Comment:
Since the return value of `self.sess_train.run(...)` is not used, please remove `_ = `.